### DISCUSSION TITLE
[Feature] [Component ZStack]

### DISCUSSION THREAD

**Discussion by Onekki** [chakra-ui/chakra-ui#8274] [Ideas]
# 🚀 Feature request

> Add component ztack (like stack in flutter)

## 🧱 Problem Statement / Justification

> Chakra UI has vstack hstack, but does not has zstack

## ✅ Proposed solution or API

> Relative postion for parent and absolue position for children

## ↩️ Alternatives

> Use grid layout

## 📝 Additional Information

> Chakra UI is a bit like flutter

**Comment by CalleEklund** [comment]
I would be happy to give this a shot!

**Comment by Onekki** [comment]
> I would be happy to give this a shot!

Looking forward to your good news!

**Comment by CalleEklund** [comment]
> > I would be happy to give this a shot!
> 
> Looking forward to your good news!

How are you thinking propwise? For example HStack and VStack has spacing affecting its children. While a ZStack would only stack the children ontop of each other?

**Comment by Onekki** [comment]
> > > I would be happy to give this a shot!
> > 
> > 
> > Looking forward to your good news!
> 
> How are you thinking propwise? For example HStack and VStack has spacing affecting its children. While a ZStack would only stack the children ontop of each other?

Yes, ZStack would only stack the children on top of each other, like ZStack in SwiftUI or Stack in Flutter.
